G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/MailImapCommand.cs

```diff
@@ -53,6 +50,9 @@ public class Arguments : CommandArguments
 
             [Argument(Required = false, Tooltip = "Received messages")]
             public VariableStructure Result { get; set; } = new VariableStructure("result");
+
+            [Argument(Required = false, Tooltip = "Ignore certificate errors")]
+            public BooleanStructure IgnoreCertificateErrors { get; set; } = new BooleanStructure(false);
```

Comment:
I don't know what to think about it, if you set IgnoreCertificateErros to false, then what happens if you still want to use this command? When the errors occur and why? Maybe if they occur always, then this boolean is useless and the errors should be **always** ignored.
@pwitusik what do you think about it?

Reply:
I introduced this variable, because I went into trouble connecting to poczta.o2.pl server. In some cases it does work with IgnoreCertificateErrors disabled properly and I don't think ignoring such errors is a good think overall and there should be an option to connect only to properly-signed servers.